source/postprocess/velocity_statistics.cc

```diff
@@ -35,34 +35,56 @@ namespace aspect
     std::pair<std::string,std::string>
     VelocityStatistics<dim>::execute (TableHandler &statistics)
     {
-      const Quadrature<dim> &quadrature_formula = this->introspection().quadratures.velocities;
-      const unsigned int n_q_points = quadrature_formula.size();
-
-      FEValues<dim> fe_values (this->get_mapping(),
-                               this->get_fe(),
-                               quadrature_formula,
-                               update_values   |
-                               update_quadrature_points |
-                               update_JxW_values);
-      std::vector<Tensor<1,dim>> velocity_values(n_q_points);
+      const QIterated<dim> quadrature_formula_trapz (QTrapezoid<1>(),
```

Comment:
```suggestion
      const QIterated<dim> quadrature_formula_trapz (QTrapezoid<1>(),
```
use a different name, maybe ``quadrature_formula_for_max`` or ``quadrature_formula_equally_spaced``